### ISSUE TITLE
3 char shields in Copenhagen are funky

### ISSUE THREAD

**Issue by sensescape** [tangrams/outdoor-style#135]

**Comment by sensescape** [comment]
<img width="680" alt="screen shot 2017-02-14 at 2 57 45 pm" src="https://cloud.githubusercontent.com/assets/466585/23081371/62732050-f509-11e6-8aa2-42df1211f27c.png">

**Comment by nvkelso** [comment]
Are the shields somehow “flat” (maybe caught up with the arrow changes)?

**Comment by sensescape** [comment]
I don't think so, this screenshot is from earlier this week, just added flat: true today. But will look into it.
